src/App.js

```diff
@@ -1,41 +1,25 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
-import logo from "./logo.png";
+import ArchivePage from "./pages/ArchivePage";
+import HomePage from "./pages/HomePage";
+import LabelPage from "./pages/LabelPage";
+import LoginPage from "./pages/LoginPage";
+import NoteTakingPage from "./pages/NoteTakingPage";
+import SignupPage from "./pages/SignupPage";
+import TrashPage from "./pages/TrashPage";
```

Comment:
Suggestion: Barrel exports will be clean

Reply:
Added barrel exports